substratevm/src/com.oracle.svm.core.genscavenge/src/com/oracle/svm/core/genscavenge/YoungGeneration.java

```diff
@@ -41,6 +43,11 @@
 
     // Final State.
     private final Space space;
+    private final SurvivorSpace[] survivorFromSpaces;
+    private final SurvivorSpace[] survivorToSpaces;
+    private final GreyObjectsWalker[] survivorGreyObjectsWalkers;
+    static final int MAX_SURVIVORS = 15;
```

Comment:
This can be a `HostedOption` instead of a constant in source code.

Reply:
done